csvkit/utilities/csvsort.py

```diff
@@ -26,7 +26,12 @@ def main(self):
             self.print_column_names()
             return
 
-        table = agate.Table.from_csv(self.input_file, sniff_limit=self.args.sniff_limit, header=not self.args.no_header_row, column_types=self.get_column_types(), **self.reader_kwargs)
+        # Otherwise, fails with "io.UnsupportedOperation: underlying stream is not seekable".
```

Comment:
Hmm, I think the merge wasn't done properly as this entire if-block shouldn't be here.